eng/pipelines/azuresdkpartnerdrops-to-devops.yml

```diff
@@ -1,22 +1,27 @@
 parameters:
-- name: PartnerDropsBlobSourceSuffix
+- name: BlobPath
```

Comment:
You can also add a displayName which contains a nice string for this something like "Relative path in partner drops blob container"